src/website/src/pages/eu/about/vision/index.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Vision
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+## Your gateway to the European Union
+
+<Paragraph size="lead">
```

Comment:
I see in the wiki they expect the font-size of the paragraph to change in mobile...